### PR TITLE
21 seeds creation

### PR DESCRIPTION
**Pull request by leezhiqin96**
# Description

Adds seed file, with comments for easy navigation
- Commented out test seed portion and placed it at the bottom of the file

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?
- [x] Test 1
Screenshot 2023-03-22 225640
- [x] Codes 
user
Programme_subject
Screenshot 2023-03-22 225130
Screenshot 2023-03-22 225154
Screenshot 2023-03-22 225212
Screenshot 2023-03-22 225218
Screenshot 2023-03-22 225227
Screenshot 2023-03-22 225231
Screenshot 2023-03-22 225244
Screenshot 2023-03-22 225246
Screenshot 2023-03-22 225255
Screenshot 2023-03-22 225257
Screenshot 2023-03-22 225307
Screenshot 2023-03-22 225312
subjects

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added proof(eg. screenshots) that prove my fix is effective or that my feature works

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1,44 +1,234 @@
...
+                                   .or(Programme.where("name ILIKE ?", "%computer%"))
+## End of career_programmes =========================================================
+
+# Soft_skills =======================================================================
+list_of_soft_skill = [
+  "Networking skills",
+  "Communication",
+  "Confidence",
+  "Organisation",
+  "Problem solving",
```
Comment (jwltay) [inline db/seeds.rb:87]: Please standardise the capitalisation of the soft skills

File: db/seeds.rb [hunk 2]
```diff
@@ -1,44 +1,234 @@
...
+  User.create!(
+    first_name: loop_name,
+    last_name: Faker::Name.last_name,
+    email: "#{loop_name}@#{['gmail', 'yahoo', 'hotmail'].sample}.#{['com', 'sg'].sample}",
+    password: "password",
+    occupation: %w[student fresh\ grad working\ adult].sample
+  )
+end
+
+User.create!(
```
Comment (jwltay) [inline db/seeds.rb:143]: Let's fix Manu's email so it's easy for us to sign in hahaha

**Review by jwltay — CHANGES_REQUESTED** [review]
Some minor changes in comments

File: db/seeds.rb (hunk 2 above)
Comment (leezhiqin96) [reply db/seeds.rb:143]: Good idea! Will do

File: db/seeds.rb (hunk 1 above)
Comment (leezhiqin96) [reply db/seeds.rb:87]: Sure thing!

**Review by jwltay — APPROVED** [review]
Looks really good!

**Pull request merged**